Sources/ArcGISToolkit/Resources/FloorFilter.storyboard

```diff
@@ -0,0 +1,254 @@
+                    </view>
+                    <extendedEdge key="edgesForExtendedLayout" bottom="YES"/>
+                    <size key="freeformSize" width="414" height="255"/>
+                    <connections>
+                        <outlet property="backBtn" destination="x20-8n-Hqp" id="F6M-mH-R2Z"/>
+                        <outlet property="bottomConstraint" destination="zxk-hd-R2q" id="PWh-w8-p15"/>
+                        <outlet property="closeBtn" destination="7Qa-bZ-qe4" id="5cB-Wl-65P"/>
+                        <outlet property="designableViewHeight" destination="nqj-n3-K3l" id="2oR-Nu-TBb"/>
+                        <outlet property="promptSubtitle" destination="sbn-5M-7Ez" id="aEm-Qe-i00"/>
+                        <outlet property="promptTitle" destination="c4S-Rd-4aZ" id="Zks-0d-wrI"/>
+                        <outlet property="promptTitleSubtitleStackView" destination="Fe7-HV-Ak8" id="9qL-JC-nNG"/>
+                        <outlet property="siteFacilitySearchBar" destination="NGq-T6-qVf" id="3mf-1o-5dl"/>
+                        <outlet property="siteFacilityTableView" destination="tgi-Um-4YG" id="rPy-Fu-W0w"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Ah8-Gw-OH4" userLabel="First Responder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="-1017" y="-196"/>
+        </scene>
+        <!--Floor Filter View Controller-->
+        <scene sceneID="xVd-Yk-Lu3">
+            <objects>
+                <viewController storyboardIdentifier="FloorFilter" autoresizesArchivedViewToFullSize="NO" automaticallyAdjustsScrollViewInsets="NO" modalPresentationStyle="currentContext" useStoryboardIdentifierAsRestorationIdentifier="YES" id="Cw7-Iz-A3J" customClass="FloorFilterViewController" customModule="ArcGISToolkit" sceneMemberID="viewController">
+                    <view key="view" opaque="NO" contentMode="redraw" layoutMarginsFollowReadableWidth="YES" insetsLayoutMarginsFromSafeArea="NO" id="xSx-j0-hTo">
+                        <rect key="frame" x="0.0" y="0.0" width="48" height="278"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="1" translatesAutoresizingMaskIntoConstraints="NO" id="hu0-iP-wgv">
+                                <rect key="frame" x="0.0" y="0.0" width="48" height="278"/>
+                                <subviews>
+                                    <button opaque="NO" contentMode="scaleAspectFit" restorationIdentifier="CloseBtn" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="6Gq-b5-cMd" userLabel="CloseBtn">
```

Comment:
If you can, please use a [system image](https://developer.apple.com/documentation/uikit/uiimage/3294233-init) instead of custom one.

Reply:
For this case, the system image for close button had a button with a shaded background. However, for the floor filter, we wanted to use a custom close button and a custom background.

Reply:
Looking at the close image you have, I believe this system symbol would work:

![image](https://user-images.githubusercontent.com/2257493/151409726-00d1d1c5-50ce-49dc-bfcc-10d95fc1dff6.png)

All symbols have a transparent background, so you should be able to use whatever background you want behind it.

Reply:
Thanks for pointing that out! Updated.